syncr_frontend/frontend.py

```diff
@@ -209,7 +210,10 @@ def get_permission(drop_id):
     owned_drops = get_owned_drops()
 
     for drop in owned_drops:
-        if drop['name'] == drop_id:
+
+        curr_id = base64.b64decode(drop.get('drop_id'), altchars=b'+-')
```

Comment:
the frontend probably shouldn't be caring about decoded drop ids

Reply:
should I just change it over to if drop.get('drop_id') == drop_id then?

Reply:
probably